### DISCUSSION TITLE
希望 gif 支持转换格式为 webp

### DISCUSSION THREAD

**Discussion by r6hk** [icret/EasyImages2.0#155] [建议]
如题，easyimage 应该是我能找到的唯一支持上传时将图片格式转换为 webp 的开源图床了（虽然有些对象存储也支持转格式为 webp，但是学生没钱不敢用啊）

gif 格式按理说也可以转换为 webp 的，但是上传 gif 之后返回的链接仍然是 gif 格式。

不好意思提 issue，所以发到 discussion 区了，希望能实现，万分感谢作者！！❤❤❤

**Comment by icret** [comment]
静态的gif是可以转换为webp的，动态的暂时不支持的

**Comment by r6hk** [comment]
> 静态的gif是可以转换为webp的，动态的暂时不支持的

好的，谢谢您

**Comment by zhendery** [comment]
顶一下，顺便问一下你提到的支持转webp的对象存储图床是什么呀（cloudflare有10G免费的对象存储，当图床够用了，不行还可以自己搭minio）
